client/deposits/list/index.js

```diff
@@ -129,17 +127,24 @@ export const DepositsList = () => {
 		return columns.map( ( { key } ) => data[ key ] || { display: null } );
 	} );
 
-	const summary = [
-		{
-			label: __( 'deposits', 'woocommerce-payments' ),
-			value: `${ depositsSummary.count }`,
-		},
-	];
-
 	const isCurrencyFiltered = 'string' === typeof getQuery().store_currency_is;
-	if ( ! isSummaryLoading ) {
-		const isSingleCurrency =
-			2 > ( depositsSummary.store_currencies || [] ).length;
+	const isSingleCurrency =
+		2 > ( depositsSummary.store_currencies || [] ).length;
+
+	let summary;
```

Comment:
Not sure about this change, why not initializing with an empty array and pushing count similar to total?

Reply:
If we do not want to render table summary then it expects a falsy value, as per https://github.com/woocommerce/woocommerce-admin/blob/b2693e66f22a8a09224ea7c97c8f6071163c06fb/packages/components/src/table/index.js#L255

If we pass an empty array (which is a truth value), then it would render the `TableSummary` component and creating a dom node that is not necessary unless we have the data.

Earlier I thought I would create a separate variable to control this behavior and pass the same to the component, something like:
```js
const summaryData = summary.length ? summary : undefined
```

But initializing `summary` with a falsy value in the first place seems like a better approach to me.

For now I'll add a comment for better clarification, please let me know what do you think?

Reply:
Great, the comment clarifies the intention really well thanks.
I'm on a final testing round then.